pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/IngestionConfig.java

```diff
@@ -115,6 +119,10 @@ public boolean isSegmentTimeValueCheck() {
     return _segmentTimeValueCheck;
   }
 
+  public boolean isSkipPartialRecords() {
```

Comment:
should we name this "skipIncompleteRecords" to align with the field "INCOMPLETE_RECORD_KEY"?